clang/test/SemaSYCL/args-size-overflow.cpp

```diff
@@ -11,9 +12,11 @@ queue q;
 using Accessor =
     accessor<int, 1, sycl::access::mode::read_write, sycl::access::target::global_buffer>;
 #ifdef SPIR64
-// expected-warning@27 {{size of kernel arguments (7994 bytes) may exceed the supported maximum of 2048 bytes on some devices}}
+// expected-warning@30 {{size of kernel arguments (7994 bytes) may exceed the supported maximum of 2048 bytes on some devices}}
+// expected-warning@31 {{size of kernel arguments (7988 bytes) may exceed the supported maximum of 2048 bytes on some devices}}
```

Comment:
I assume the difference in size is due to difference in __init and __init_esimd function arguments?

Reply:
Yes, that is my understanding.